phpunit.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should not be added, instead changes should be made to `phpunit.xml.dist`